src/main/java/uk/gov/ons/ctp/response/action/service/impl/ActionServiceImpl.java

```diff
@@ -164,11 +177,40 @@ public Action createAction(final Action action) {
 
     action.setManuallyCreated(true);
     action.setCreatedDateTime(DateTimeUtil.nowUTC());
-    action.setState(ActionDTO.ActionState.SUBMITTED);
+    action.setState(ActionState.SUBMITTED);
     action.setId(UUID.randomUUID());
     return actionRepo.saveAndFlush(action);
   }
 
+  @Transactional
+  @Override
+  public boolean createScheduledActions(Integer actionPlanJobId) {
+    final ActionPlanJob actionPlanJob =
+        actionPlanJobRepository.findByActionPlanJobPK(actionPlanJobId);
+
+    if (actionPlanJob == null) {
+      return true;
```

Comment:
Should this be 'false'? Otherwise this method seems to only return true

Reply:
It does only return true, this was the behaviour of the stored proceedure which is why it's still in here. We could remove it though.

Reply:
Removed